Ghost-Ashu.md

```diff
@@ -0,0 +1,34 @@
+---
+name: "@Ghost-Ashu"
+project: "RealTime GPS Data Logger with GMaps"
+---
+---
+#### Email: ashutoshkumarsingh951@gmail.com
+---
+# RealTime GPS Data Logger with Goggle Maps
+
+## Summary
+
+Have You ever wonder! How Gobal Positioning System Tracks and Logs our Cordinates. Here  is my Project Walkthrough, where i will be building a realtime GPS Data Logger by using  Google Maps API and NodeMCU ESP8266.
+
+## Plan
+
+The Latitude and Longitude on the Display of 3.5inch TFT SPI LCD(ILI9488). It'll act as interface with GPS Module and NodeMCU ESP8266 also it will display the GPS data (Latitude, longitude, date and time). I have Tried to Collect from all the Sources in Building my Project
+ESP8266 Showing the Device position on the Display Screen. Using the Google Cloud platform i will be using the Static API of G- Maps. Display the geographic location of a user or device on a Google map.
+
+## Circuit And Architecture
+![20221224_124036 (2)](https://user-images.githubusercontent.com/41056892/209425577-d997ff2b-fc51-44ff-b319-5751af7988ad.png)
+
+
+## Budget
+
+| Product                                          | Supplier/Link                                                                   | Cost   |
+| -------------------------------------------------| --------------------------------------------------------------------------------| ------ |
+| Arduino Uno Rev3                                 |     https://store.arduino.cc/products/arduino-uno-rev3                          | $25.60 |
```

Comment:
It's not quite clear in your project summary/plan what this Arduino is for. Could you clarify?